doc/design.txt

```diff
@@ -210,15 +210,15 @@ An image trailer has the following structure:
      0                   1                   2                   3
```

Comment:
I don't think it does. I know what you mean, but there are two ways of explaining things in the design doc: one more "abstract", which uses words like `pending` and `confirmed` and the practical, real implementation, which uses `magic`, `image_ok` and `copy_done`.